### ISSUE TITLE
FirewallRule Logging

### ISSUE THREAD

**Issue by haribcs** [vmware/powernsx#410]
I have multiple firewall rules , I have a list of few rules logging has to be enabled for syslog servers. Could please help to get the command for it?

I tried 
$rule = get-nsxfirewallrule -ruleid XXX
$rule.logging = "true"
However it is not reflecting the in the firewall rule.

**Comment by alagoutte** [comment]
you need to "apply" change after...

but there is not yet set-nsxfirewallrule function :(

**Comment by nmbradford** [comment]
@alagoutte, there is always more than one way to skin a cat! :)

@haribcs - take a look at https://github.com/vmware/powernsx/blob/master/Examples/EnableFirewallRuleLogging.ps1

It is an example of enabling logging on all rules in the DFW.  You obviously want to be more selective about what is, so you will need to modify it for your purposes, but it shows the steps necessary and modifying the loop on line 45 should be pretty easy to accomplish what you are after.

A word of warning : please _make sure_ you test your changes in a non-production environment before running this in production, a wrong step here could destroy your rulebase.

Issue #405 is already raised to track this.

**Comment by alagoutte** [comment]
@nmbradford Yes, it is true ! but it is more easy with direct PowerNSX function :) i missing the issue #405 ... (it is funny because there is already a reference of set-nsxfirewallrule on PowerNSX for a internal comment !)
